### PR TITLE
Update BLS library

### PR DESCRIPTION
**Pull request by Giulio2002**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/erigontech/erigon
 
-go 1.22.0
+go 1.23
```
Comment (AskAlexSharov) [inline go.mod:3]: probably this change not related to PR?

**Review by AskAlexSharov — CHANGES_REQUESTED** [review]

**Comment by AskAlexSharov** [comment]
FYI: it solves the problem

File: go.mod (hunk 1 above)
Comment (Giulio2002) [reply go.mod]: I think the toolchain is tho

**Comment by Giulio2002** [comment]
fixes https://github.com/erigontech/erigon/issues/12287

File: go.mod (hunk 1 above)
Comment (Giulio2002) [reply go.mod]: actually no, it does not :)

**Review by AskAlexSharov — APPROVED** [review]

**Pull request merged**